src/config/fixtures/default.toml

```diff
@@ -0,0 +1,43 @@
+# Branch (required)
+# Regex to choose which branch(-es) this block applies to, some examples
+# ["will\\/.*"] applies to branches like`will/release-workflows`
+# ["^\\d+\\/.*"] applies to branches starting with numbers `123/feat-some-issue`
+# [main] applies only to the branch called `main`
+# ["*"] is the reserved default that applies if no block of higher precedence is present.
+# ["*"] It also applies if current folder is not tracked under source control
+["*"]
+
+# Workspace (optional, default=null)
+# in this context would resolve to the user's default workspace
+# https://support.toggl.com/en/articles/2452474-introduction-to-workspaces#switching-and-creating-workspaces
+# workspace = "My Workspace"
+
+# Description (optional, default="")
+# Accepts any string template with our marcros in them
+description = "🔨 {{current_branch}}"
+
+# Project (optional, default=No project)
+# Accepts any string template with our marcros in them
+# A project is created if it doesn't exist
```

Comment:
That may fail if the user might not have permission in the workspace, wil just need to handle the rejected request: https://support.toggl.com/en/articles/2216605-access-rights-and-privileges#:~:text=A%20regular%20user%20can%3A&text=Add%20projects%20and%20clients%20(unless,Add%20tags. Or could also support a mapping/remember the project last assigned to a directory (similar to official browser extension)

Reply:
Good point, will track for when we implement the last bit that actually ensures all entities are present and creates the time-entry from the config.

Reply:
> Or could also support a mapping/remember the project last assigned to a directory (similar to official browser extension

That would be achieved it they manually add the project name (of one that already exists)
```toml
project = "toggl-cli"
```

or are you suggestion another template variable they could utilize instead?

Reply:
If we're not creating the entities yet we should remove the comment from the default toml

Reply:
Dropped the comment.